lib/rucio/rse/protocols/globus.py

```diff
@@ -14,38 +13,52 @@
 # limitations under the License.
 #
 # Authors:
-# - Matt Snyder <msnyder@bnl.gov>, 2019-2020
+# - Matt Snyder <msnyder@bnl.gov>, 2019-2021
 # - Martin Barisits <martin.barisits@cern.ch>, 2019
-# - Tomas Javurek <tomas.javurek@cern.ch>, 2020
-# - Benedikt Ziemons <benedikt.ziemons@cern.ch>, 2021
 
 from __future__ import print_function
 
-from six import string_types
-from six.moves.urllib.parse import urlparse
+import imp
+import logging
 
+from six import string_types
 from rucio.common import exception
-from rucio.common.extra import import_extras
 from rucio.core.rse import get_rse_attribute
 from rucio.rse.protocols.protocol import RSEProtocol
-from rucio.transfertool.globusLibrary import getTransferClient, send_delete_task
+from rucio.transfertool.globus_library import get_transfer_client, send_delete_task, send_bulk_delete_task
+
+# Extra modules: Only imported if available
+EXTRA_MODULES = {'globus_sdk': False}
 
-EXTRA_MODULES = import_extras(['globus_sdk'])
+for extra_module in EXTRA_MODULES:
```

Comment:
@bziemons adapted this in a PR, it should be done similarly. See #4168

Reply:
On it.  I also see I reverted lib/rucio/transfertool/globus-library.py I will fix that as well. That is some slick code @bziemons !

Reply:
@maatthias Thanks! It would be great if you can make use of it in `lib/rucio/transfertool/globus_library.py` as well :)